src/main/java/org/reactome/release/cosmicupdate/Main.java

```diff
@@ -57,26 +64,49 @@ public static void main(String... args)
 	{
 		try
 		{
-			try(Reader configReader = new FileReader("src/main/resources/config.properties"))
+			boolean noConfigSpecified = Main.configPath != null && !Main.configPath.trim().isEmpty();
+//			boolean noAuthConfigSpecified = Main.authPath != null && !Main.authPath.trim().isEmpty();
+			if (!noConfigSpecified /* && !noAuthConfigSpecified */)
+			{
+				try(Reader configReader = new FileReader(Main.configPath);
+				/* Reader authReader = new FileReader(Main.authPath) */)
+				{
+					Properties configProps = new Properties();
+					configProps.load(configReader);
+					
+//					Properties authProps = new Properties();
+//					authProps.load(authReader);
+
+					// These default to files in the same directory where the program is running.
+					Main.COSMICMutantExport = configProps.getProperty("pathToMutantExportFile", "./CosmicMutantExport.tsv");
+					Main.COSMICFusionExport = configProps.getProperty("pathToFusionExportFile", "./CosmicFusionExport.tsv");
+					Main.COSMICMutationTracking = configProps.getProperty("pathToMutationTrackingFile", "./CosmicMutationTracking.tsv");
+					// These have NO default.
+					Main.COSMICUsername = configProps.getProperty("cosmic.username");
```

Comment:
In addlinks, I believe it actually uses the property name `cosmic.user` - could these property names get synced up?